.babelrc

```diff
@@ -40,6 +40,7 @@
   ],
   "plugins": [
     "lodash",
+    "istanbul",
```

Comment:
Can istanbul be moved to the `test` environment?

Reply:
I tried placing istanbul in the `test` environment but then the code coverage report would not generate during the GHA workflow.